threefoldtech/tfgrid-sdk-ts
# Add mycelium IP column in workers table

### Description

- Add the MyCelium IP column in the worker's table.
- Include myCelium in the worker deployment object.
- Add `Created At` and `Health` columns.

### Changes

[Screencast from 25-11-24 16:21:46.webm](https://github.com/user-attachments/assets/74aa2020-8440-4830-9840-4921c18d2864)

### Tested Scenarios

- Go to `Caprover` solution
- Deploy instance with worker and mycelium enabled
- List the deployments 
- View the worker's table, the mycelium column should be there

### Scenarios update

- Create more than two workers and check that all have the mycelium IP column.
- Create with planetary and no mycelium and check nothing in the mycelium column.
- Delete a worker, add it again, and ensure mycelium ip changes.

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3446

### Checklist
